### PR TITLE
Blog post for EO and the recent OMB memo

### PR DESCRIPTION
**Pull request by hepwori**
Adds a blog post with an educational plain-english summary of the implications of the recent OMB memo following the Executive Order on Improving the Nation’s Cybersecurity. 

Signed-off-by: Isaac Hepworth <isaac.hepworth@gmail.com>

### CODE REVIEW

**Comment by kimsterv** [comment]
lgtm

File: docs/_posts/2022-09-26-eo-in-plain-english.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+3. Read up on SBOMs. A 2019-era NTIA whitepaper “[Roles and Benefits for SBOM Across the Supply Chain](https://www.ntia.gov/files/ntia/publications/ntia_sbom_use_cases_roles_benefits-nov2019.pdf)” includes a comprehensive catalog of potential benefits offered by SBOMs, and the companion document [Minimum Elements of an SBOM](https://www.ntia.doc.gov/files/ntia/publications/sbom_minimum_elements_report.pdf) lays out a pragmatic starting point.
+4. Review [NIST’s definition](https://www.nist.gov/itl/executive-order-improving-nations-cybersecurity/critical-software-definition-explanatory) of “critical software” to understand if your products fall into this category. If so, your conformance with the new standards is expected by June next year, rather than September.
+5. See how SLSA can help! Read the previous posts [SBOM + SLSA: Accelerating SBOM success with the help of SLSA](https://slsa.dev/blog/2022/05/slsa-sbom) and [SLSA for Success: Using SLSA to help achieve NIST’s SSDF](https://slsa.dev/blog/2022/06/slsa-ssdf)
```
Comment (joshuagl) [inline docs/_posts/2022-09-26-eo-in-plain-english.md:64]:
```suggestion
5. See how SLSA can help! Read the previous posts [SBOM + SLSA: Accelerating SBOM success with the help of SLSA](https://slsa.dev/blog/2022/05/slsa-sbom) and [SLSA for Success: Using SLSA to help achieve NIST’s SSDF](https://slsa.dev/blog/2022/06/slsa-ssdf).
```

this list item was the only one that didn't use a period.

File: docs/_posts/2022-09-26-eo-in-plain-english.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+*   Secure Software Development Framework (SSDF), SP 800-218 [[link](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-218.pdf)]
+*   Software Supply Chain Security Guidance [[link](https://www.nist.gov/system/files/documents/2022/02/04/software-supply-chain-security-guidance-under-EO-14028-section-4e.pdf)]
+
+We covered the SSDF briefly in a [previous slsa.dev blog post](https://slsa.dev/blog/2022/06/slsa-ssdf), and noted how SLSA could form a ready on-ramp to successful SSDF conformance.
+
+
+## What’s new
+
+Last week the White House Office of Management and Budget (OMB) issued [a memorandum](https://www.whitehouse.gov/wp-content/uploads/2022/09/M-22-18.pdf) requiring every Federal agency to comply with the NIST Guidance “when using third-party software on the agency’s information systems or otherwise affecting the agency’s information”.
```
Comment (joshuagl) [inline docs/_posts/2022-09-26-eo-in-plain-english.md:30]: Perhaps better to use a date, rather than "last week" – who knows when folks are reading.

**Review by joshuagl — APPROVED** [review]
Nice work.

File: docs/_posts/2022-09-26-eo-in-plain-english.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Executive Order on Secure Supply Chain — in Plain English"
+author: "Isaac Hepworth"
+is_guest_post: true
+---
+
+You probably heard about [EO 14028](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/), the “Executive Order on Improving the Nation’s Cybersecurity”.
+
+Last week the White House Office of Management and Budget (OMB) issued a memorandum setting firm and aggressive timelines for implementation of guidelines stemming from the EO, and you might reasonably be wondering what it all means. If so, this post is for you. We’re going to try to lay it out in plain English and share steps to help you get ready to meet the timelines
```
Comment (MarkLodato) [inline docs/_posts/2022-09-26-eo-in-plain-english.md]: nit: link to the memorandum here?

File: docs/_posts/2022-09-26-eo-in-plain-english.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Executive Order on Secure Supply Chain — in Plain English"
+author: "Isaac Hepworth"
+is_guest_post: true
+---
+
+You probably heard about [EO 14028](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/), the “Executive Order on Improving the Nation’s Cybersecurity”.
```
Comment (MarkLodato) [inline docs/_posts/2022-09-26-eo-in-plain-english.md:7]: suggestion: "You probably heard about" may put off readers who have not heard in fact heard about it or who don't know the details. Instead, consider using "You may have" and then adding a brief explanation of what it is, e.g.:

"You may have heard about ..., which sets minimum supply chain security standards for all software consumed by the US government."

File: docs/_posts/2022-09-26-eo-in-plain-english.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+After a [request for comments](https://www.ntia.gov/files/ntia/publications/frn-sbom-rfc-06022021.pdf) in **June 2021**, the US Department of Commerce [issued in July 2021](https://www.ntia.gov/blog/2021/ntia-releases-minimum-elements-software-bill-materials) a report [Minimum Elements for an SBOM](https://www.ntia.doc.gov/files/ntia/publications/sbom_minimum_elements_report.pdf), as part of establishing new baseline security standards for development of software used by the US Federal Government. We published a [blog post on SLSA and SBOMs](https://slsa.dev/blog/2022/05/slsa-sbom) a little while ago.
```
Comment (MarkLodato) [inline docs/_posts/2022-09-26-eo-in-plain-english.md:18]: nit: use relative links to blog posts throughout the article, e.g. `[blog post on SLSA and SBOMs](2022-05-02-slsa-sbom.md)`.

File: docs/_posts/2022-09-26-eo-in-plain-english.md (hunk 2 above)
Comment (hepwori) [reply docs/_posts/2022-09-26-eo-in-plain-english.md]: Ack, good catch thanks. Updated.

File: docs/_posts/2022-09-26-eo-in-plain-english.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (MarkLodato) [inline docs/_posts/2022-09-26-eo-in-plain-english.md:1]: Please run `npm run format` (or `npm run lint`) and fix any issues identified. For example, extra blank lines.

File: docs/_posts/2022-09-26-eo-in-plain-english.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+In **September 2022** the White House Office of Management and Budget (OMB) issued [a memorandum](https://www.whitehouse.gov/wp-content/uploads/2022/09/M-22-18.pdf) requiring every Federal agency to comply with the NIST Guidance “when using third-party software on the agency’s information systems or otherwise affecting the agency’s information”.
+
+In **June 2023**, compliance with the NIST Guidance will be required for all “critical software” used by Federal agencies.
+
+In **September 2023**, compliance with the NIST Guidance will be required for all third-party software used by Federal agencies.
+
+
+## What it means
+
+If your software is sold to, or used by, the US Federal government then your ears should be pricking up about now. Starting next year you will be expected, for each major version of each software product you supply, to provide:
```
Comment (MarkLodato) [inline docs/_posts/2022-09-26-eo-in-plain-english.md:38]: optional: You might want to add a second paragraph noting that even if this doesn't apply to you now, there's a decent chance your customers will start demanding the same thing within the next few years, so you may want to get ahead of the game.

**Review by MarkLodato — CHANGES_REQUESTED** [review]
Nice! A few style nits but overall content is good.

File: docs/_posts/2022-09-26-eo-in-plain-english.md (hunk 6 above)
Comment (MarkLodato) [reply docs/_posts/2022-09-26-eo-in-plain-english.md:1]: Or just see the [failing check](https://github.com/slsa-framework/slsa/actions/runs/3137676841/jobs/5096147744) if you don't feel like running npm.

File: docs/_posts/2022-09-26-eo-in-plain-english.md (hunk 3 above)
Comment (hepwori) [reply docs/_posts/2022-09-26-eo-in-plain-english.md]: Done!

File: docs/_posts/2022-09-26-eo-in-plain-english.md (hunk 5 above)
Comment (hepwori) [reply docs/_posts/2022-09-26-eo-in-plain-english.md]: 👍

File: docs/_posts/2022-09-26-eo-in-plain-english.md (hunk 7 above)
Comment (hepwori) [reply docs/_posts/2022-09-26-eo-in-plain-english.md:34]: 👍

**Review by MarkLodato — APPROVED** [review]

File: docs/_posts/2022-09-26-eo-in-plain-english.md (hunk 6 above)
Comment (hepwori) [reply docs/_posts/2022-09-26-eo-in-plain-english.md:1]: Ack! Passes the linter now. Thanks :)
